browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3047,10 +3047,7 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       Top Visited Site
     </message>
     <message name="IDS_SEARCH_WITH_BRAVE" desc="String used in QuickActionSearchAndBookmarkWidget's search bar">
-      Search with Brave
-    </message>
-    <message name="IDS_SEARCH_WITH_SEARCH_ENGINE" desc="Dynamic string used in QuickActionSearchAndBookmarkWidget's search bar. Placeholder is replaced by default search engine.">
-      Search with <ph name="DEFAULT_SEARCH_ENGINE">%1$s</ph>
+      Search
```

Comment:
we decided to just do `Search` as `Search with Brave` doesn't fit and doesn't look good on some small screens.